pom.xml

```diff
@@ -72,6 +72,12 @@
                     <include>**/*.properties</include>
                 </includes>
             </resource>
+            <resource>
```

Comment:
The license info files are packaged into jar file:
![image](https://github.com/parasoft/parasoft-findings-utils/assets/109122571/15f691a0-9d4d-44a2-a18b-4981b87cc8ea)